molecule/resources/verify.yml

```diff
@@ -36,7 +36,8 @@
             openwisp_controller.config.tests.test_admin.TestAdmin.test_vpn_not_contains_default_templates_js \
             openwisp_controller.config.tests.test_notifications \
             openwisp_controller.geo.tests.test_api \
-            openwisp_controller.subnet_division.tests.test_models
+            openwisp_controller.subnet_division.tests.test_models \
+            openwisp_notifications.tests.test_admin
```

Comment:
I think these tests are failing because of the static minification.

We could separate these two tests:
https://github.com/openwisp/openwisp-notifications/blob/master/openwisp_notifications/tests/test_admin.py#L253-L297

In a different class called `TestAdminMedia` and here we can put `openwisp_notifications.tests.test_admin.TestAdmin`, that would do.

Reply:
@nemesisdesign  Done :+1:  https://github.com/openwisp/openwisp-notifications/pull/227